src/inc/corinfo.h

```diff
@@ -1839,12 +1839,28 @@ enum { LCL_FINALLY_MARK = 0xFC }; // FC = "Finally Call"
 
 #include <pshpack4.h>
 
+#ifndef CROSS_COMPILE
```

Comment:
I think it would be better to either delete these structs; or put them under a big ifdef if you need to keep them around to keep the legacy jits building.

And then just define constants for the offsets that the JIT needs - like we are doing in other places.

Alternatively, these offsets can be returned over JIT/EE interface - like it is done with `offsetOfObjArrayData`. I am not sure whether it is worth it unless there is a good reason to. Either way is fine with me though.

Reply:
@jkotas I re-did as you suggested - defined constants, replaced the usage of `offsetof(CORINFO_X, Y)` in clrjit
What macro do you think I should use to guard CORINFO structures?

Reply:
@briansull Do we need these headers still work for the internal desktop builds without any changes?

If yes, we should put them under a define that is only defined for the internal desktop build. We can create a new one or piggy back on some existing one - cannot think about a good candidate.
If no, you can just delete these.